.eslintrc.json

```diff
@@ -0,0 +1,44 @@
+{
+    "env": {
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended",
+        "airbnb",
+        "airbnb/hooks",
+        "import",
+        "prettier",
+        "prettier/react"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint",
+        "react",
+        "react-native"
+    ],
+    "rules": {
+        "indent": [
```

Comment:
Te rule odnośnie tego no można dać, ale imo to zbędne, lepiej według mnie mieć to prettierem ograne i mieć pipeline (gh actions here), który sprawdzi style niż w eslint to deklarować imo. Poza tym, te extendsy co dałem wyżej też ci to załatwią. 

Mogę ci taki mój ulubiony eslint konfig podesłać jak chcesz xd